admin_guide/high_availability.adoc

```diff
@@ -581,30 +587,38 @@ $ oadm router ha-router-us-west --replicas=5 \
 ====
 endif::[]
 +
-.. Run the *geo-cache* service with a replica on each of the nodes. An example configuration for running a *geo-cache* service https://raw.githubusercontent.com/openshift/openshift-docs/master/admin_guide/examples/geo-cache.json[is
+.. Run the *geo-cache* service with a replica on each of the nodes. An example
+configuration for running a *geo-cache* service
+https://raw.githubusercontent.com/openshift/openshift-docs/master/admin_guide/examples/geo-cache.json[is
 provided here].
```

Comment:
Maybe rewrite this to avoid the "here" statement. Something like:

    See an link:https://raw.githubusercontent.com/openshift/openshift-docs/master/admin_guide/examples/geo-cache.json[example configuration] for running a *geo-cache* service.